packages/catala/catala.0.7.0/opam

```diff
@@ -0,0 +1,80 @@
+opam-version: "2.0"
+synopsis:
+  "Compiler and library for the literate programming language for tax code specification"
+description:
+  "Catala is a domain-specific language for deriving faithful-by-construction algorithms from legislative texts. See https://catala-lang.org for more information"
+maintainer: ["contact@catala-lang.org"]
+authors: [
+  "Denis Merigoux"
+  "Nicolas Chataing"
+  "Emile Rolley"
+  "Louis Gesbert"
+  "Aymeric Fromherz"
+  "Alain Delaët-Tixeuil"
+]
+license: "Apache-2.0"
+homepage: "https://github.com/CatalaLang/catala"
+bug-reports: "https://github.com/CatalaLang/catala/issues"
+depends: [
+  "ANSITerminal" {>= "0.8.2"}
+  "benchmark" {>= "1.6"}
+  "bindlib" {>= "5.0.1"}
+  "calendar" {>= "2.04"}
+  "cmdliner" {>= "1.1.0"}
+  "cppo" {>= "1"}
+  "dune" {>= "2.8"}
+  "js_of_ocaml-ppx" {>= "4.0.0"}
+  "menhir" {>= "20200211"}
+  "menhirLib" {>= "20200211"}
+  "ocaml" {>= "4.13.0"}
+  "ocamlgraph" {>= "1.8.8"}
+  "ppx_yojson_conv" {>= "0.14.0"}
+  "re" {>= "1.9.0"}
+  "sedlex" {>= "2.4"}
+  "ubase" {>= "0.05"}
+  "unionFind" {>= "20200320"}
+  "visitors" {>= "20200210"}
+  "zarith" {>= "1.12"}
+  "zarith_stubs_js" {>= "v0.14.1"}
+  "alcotest" {with-test & >= "1.5.0"}
+  "odoc" {with-doc}
+  "ocamlformat" {cataladevmode & = "0.21.0"}
+  "obelisk" {cataladevmode}
+  "conf-npm" {cataladevmode}
+  "conf-python-3-dev" {cataladevmode}
+  "z3" {catalaz3mode}
```

Comment:
This is breaking builds for me with the error:

> undefined filter variable in dependencies of catala.0.7.0: catalaz3mode

Reply:
@altgr?

Reply:
Hmm, could you explicit the opam version you're using, and what specific options ? E.g. maybe `--strict` ?

Reply:
(it would be fine to remove these dependencies for the opam repository, while keeping them in our source for devs — we expect to leverage the upcoming `for-dev` feature once it's available — but the undefined variables shouldn't stop opam from continuing in general)

Reply:
2.0.7 and yes I'm using strict

Reply:
Similar issue here. My build breaks with --strict on this error, but without --strict it breaks on a "No agreement on the version of ocaml" error instead. Not sure if related.

Reply:
This should be fixed in https://github.com/ocaml/opam-repository/pull/21996. cc @na4zagin3 too